Dockerfile

```diff
@@ -2,25 +2,25 @@ FROM node:6.10.3-alpine
 
 WORKDIR /app
 
-COPY package.json package.json
-
 RUN apk --no-cache update && \
     apk --no-cache upgrade && \
     apk add --no-cache fontconfig && \
-    apk add --no-cache --virtual .build-dependencies curl git && \
-    curl -Ls "https://github.com/tidepool-org/tools/raw/master/alpine_phantomjs_dependencies/dockerized-phantomjs.tar.xz" | tar xJ -C / && \
-    mkdir -p dist node_modules /@tidepool/viz/node_modules /tideline/node_modules /tidepool-platform-client/node_modules && \
-    chown -R node:node . /@tidepool /tideline /tidepool-platform-client && \
-    apk del .build-dependencies && \
-    rm -rf /usr/share/man /tmp/* /var/tmp/* /root/.npm /root/.node-gyp
+    apk add --no-cache --virtual .build-dependencies curl git
 
-USER node
+COPY package.json package.json
+RUN npm install npm@6
+
+RUN chown -R node:node .
 
-RUN yarn install && \
-    yarn cache clean
+USER node
+RUN mkdir -p dist node_modules
+COPY ./node_modules/@tidepool ./node_modules
```

Comment:
is it still necessary to copy those modules? 
I would think it is managed by the npm install, line 11
Same for next line, line 18

Reply:
I left this for the future, when (someday) we will retrieve tideline and viz from Nexus, but you're right, let's delete this and re-add later when required.